protected/config/core/common.php

```diff
@@ -303,7 +303,7 @@
                 'uri' => '#',
                 'position' => 2,
                 'userrule' => 'isSurgeon',
-                'restricted' => array('admin', 'Report', 'NOD Export', 'Add patient', 'Patient Merge', 'Patient Merge Request', 'Optom co-ordinator'),
+                'restricted' => array('admin', 'Report', 'NOD Export', 'Add patient', 'Patient Merge', 'Patient Merge Request', 'Optom co-ordinator', 'View Practitioner', 'Manage Practitioner', 'View Practice', 'Manage Practice'),
```

Comment:
I suspect that this is done with hierarchical RBAC in mind. Using the roles encompasses all the tasks that inherit from that role.